src/telemetry/graph.js

```diff
@@ -7,17 +7,15 @@ import SETTINGS from '../settings';
 
 export default class TelemetryContainer extends React.Component {
   async componentDidMount() {
-    const { id } = this.props;
-    this.fetchPlotGraph(id);
+    this.fetchPlotGraph(this.props.id, this.props.queryParams);
   }
 
-  async fetchPlotGraph(id) {
-    const { queryParams } = this.props;
-    let url = `${SETTINGS.backend}/api/perf/telemetry/${id}`;
-
-    if (Object.keys(queryParams).length > 0) {
-      url += `?${stringify(queryParams)}`;
-    }
+  async fetchPlotGraph(id, queryParams) {
+    let url = `${SETTINGS.backend}/api/perf/telemetry?`;
+    url += stringify({
+      name: id,
+      ...queryParams,
```

Comment:
Why would you need to pass in a copy of queryParams? Does stringify mutate it? I was trying to test this locally but hit CORS issues with the temp backend.

Reply:
You either need to `yarn start:local` or use the BACKEND env variable to point to https://firefox-health-backend-pr-32.herokuapp.com

I don't exactly know what you're referring.

Javascript's object destructuring is similar to doing this:
```
const fooObject = { name: id };
Object.entries(queryParams).forEach((key, value) => fooObject[key] = value);
```
Also, as part of the linting rules, I am not allowed to modify any parameters specified on a function's signature. In our case `id` and `queryParams` (referred signature: `fetchPlotGraph(id, queryParams)`).

Reply:
Sorry for the confusing statement - I thought I had edited my comment.  I did get the server running locally in the end. I initially had a strange caching issue with the BACKEND env variable change not being bundled and rendered despite it clearly showing up in the code.

As for the queryParams question, what I meant was, I was curious why you needed to use spread syntax  `...queryParams` - to copy properties of that object - rather than just passing the original object in to stringify. Does stringify mutate the original object?